### PR TITLE
1749 Enable Serving WMS for 3D Databases

### LINKED ISSUE #1749: Allow users to serve WMS of 3D files from server UI
It was previously believed that serving WMS was unusable to obtain tiles from 3D portable files. However, it was recently found that serving WMS on a 3D portable file will still send 2D tile information to WMS clients.

3D Databases can be published with --serve_wms, but there should be an option in the UI for doing it as well.

### PR DESCRIPTION
**Pull request by tst-nrichman**
All of the back-end work to publish databases with the proper WMS serve is already in place. This change simply makes the button visible in the geserver user interface.

Fixes #1749

### CODE REVIEW

File: earth_enterprise/src/server/wsgi/wms/ogc/xml/v130/capabilities_wms.py [hunk 1]
```diff
@@ -2335,12 +2335,12 @@ class Layer(GeneratedsSuper):
     subclass = None
     superclass = None
     def __init__(self, opaque=False, cascaded=None, fixedHeight=None, fixedWidth=None, noSubsets=False, queryable=False, Name=None, Title=None, Abstract=None, KeywordList=None, CRS=None, EX_GeographicBoundingBox=None, BoundingBox=None, Dimension=None, Attribution=None, AuthorityURL=None, Identifier=None, MetadataURL=None, DataURL=None, FeatureListURL=None, Style=None, MinScaleDenominator=None, MaxScaleDenominator=None, Layer=None):
-        self.opaque = _cast(bool, opaque)
+        self.opaque = _cast(int, opaque)
```
Comment (tst-blaird) [inline earth_enterprise/src/server/wsgi/wms/ogc/xml/v130/capabilities_wms.py:2338]: What's the reason for changing the cast of opaque, noSubsets, and queryable to int?  Looking at the file they are set to bool values of True/False.

Comment (tst-nrichman) [reply earth_enterprise/src/server/wsgi/wms/ogc/xml/v130/capabilities_wms.py:2338]: Trying to integrate our WMS serving with OWSLib. The library expects our responses to return integers. So it's fine to set true/false throughout, but the actual value will need the cast.

Comment (tst-blaird) [reply earth_enterprise/src/server/wsgi/wms/ogc/xml/v130/capabilities_wms.py:2338]: ok, that works.  Thanks for the explanation.

**Comment by tst-ccamp** [comment]
Make sure to test this with portable globes. WMS will not work for them.

**Comment by tst-hrajamoney** [comment]
Tested WMS on 3d globes and 2d maps and 2d portables. Code change looks good.

**Pull request merged**